conjurapi/logging/logging.go

```diff
@@ -0,0 +1,35 @@
+package logging
+
+import (
+	"io"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+var ApiLog = logrus.New()
```

Comment:
Can `const` be used here?

Reply:
I don't think so. The only way to get a new instance of a logger is using `logrus.New()`, which can't be evaluated at compile time.